tests/emscripten_api_browser.c

```diff
@@ -105,7 +104,7 @@ int main() {
 
   assert(ratio == ratio2);
 
-  atexit(never); // should never be called - it is wrong to exit the runtime orderly if we have async calls!
+  atexit(check_exit_ok);
```

Comment:
The previous code says this should never be called.. Is this still not supposed to be called? If so it'd be good to make that clear in the comments. If not what changed?